docs/sdk/src/guides/your_first_pallet/mod.rs

```diff
@@ -160,13 +161,12 @@
 #![doc = docify::embed!("./src/guides/your_first_pallet/mod.rs", first_test)]
 //!
 //! In the first test, we simply assert that there is no total issuance, and no balance associated
-//! with account `1`. Then, we mint some balance into `1`, and re-check.
+//! with ALICE's account. Then, we mint some balance into ALICE's, and re-check.
```

Comment:
not by any means a hill I'd die on but I feel this reads nicer when not shouting ALICE 😂

```suggestion
//! with Alice's account. Then, we mint some balance into Alice's, and re-check.
```